config/locales/en.yml

```diff
@@ -934,6 +943,7 @@ en:
         could_not_be_copied: "%{dependency} could not be (fully) copied."
         does_not_exist: "does not exist."
         error_enterprise_only: "%{action} is only available in the OpenProject Enterprise edition"
+        error_not_found: "not found."
```

Comment:
🟡 prefixing the locale key with `error` again seems ... duplicate? Could we use `not_found` instead?